README.md

```diff
@@ -3,6 +3,8 @@
 [![Code Climate](https://codeclimate.com/github/instructure/lti2_reference_tool_provider.svg)](https://codeclimate.com/github/instructure/lti2_reference_tool_provider)
 [![Dependency Status](https://gemnasium.com/badges/github.com/instructure/lti2_reference_tool_provider.svg)](https://gemnasium.com/github.com/instructure/lti2_reference_tool_provider)
 
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```

Comment:
on the readme page there will be this button: ![Deploy](https://www.herokucdn.com/deploy/button.svg)

When they click it it will launch to heroku, ask them to log in, and then give them the option of deploying the app to their heroku account.